libiop/protocols/encoded/lincheck/basic_lincheck_aux.hpp

```diff
@@ -56,6 +56,8 @@ class multi_lincheck_virtual_oracle : public virtual_oracle<FieldT> {
     lagrange_polynomial<FieldT> p_alpha_;
     std::vector<FieldT> p_alpha_evals_;
     std::vector<FieldT> p_alpha_ABC_evals_;
+    vanishing_polynomial<FieldT> vd_vp_;
```

Comment:
Can vd and cd have their names expanded? (We haven't been abbreviating the domain names in the codebase, if we want to change it we should do it all at once. Currently unclear to me if we want to or not)